tests/erfcx_test.rs

```diff
@@ -0,0 +1,153 @@
+//! This test compares the Rust port's output with the output from
+//! the following C program:
+//!
+//! ```C
+//! #include <cerf.h>
+//! #include <stdio.h>
+//! #include <complex.h>
+//!
+//! int main() {
+//!     double x = 0.0;
+//!     for (int i = 0; i < 1000; i++) {
+//!         x += 0.1;
+//!         double res = erfcx(x);
+//!         printf("%17lf,\n", res);
```

Comment:
```suggestion
//!         printf("%.17e,\n", res);
```

You don't specify the precision, so only use 6 digits.

> If the precision is missing, it shall be taken as 6;
> https://stackoverflow.com/questions/1207762/what-is-c-printf-f-default-precision

This gives:
```
$ gcc -o erfcx erfcx.c -lcerf -lm && ./erfcx
8.96456979969126766e-01,
8.09019519901580830e-01,
7.34599334567655360e-01,
6.70787785294761463e-01,
6.15690344192925787e-01,
5.67804717386586999e-01,
...
```

Reply:
Ah, I was missing the period. I've changed that, but you're right, scientific notation might be a better idea.